### DISCUSSION TITLE
Obtain original points from detect_planar_patches()

### DISCUSSION THREAD

**Discussion by Antjubu** [isl-org/Open3D#6656] [Q&A]
The method detect_planar_patches() returns the bounding boxes of detected planes from a point cloud. However, I would like to perform further processing of my data with the actual points that form the planar patches.
Is there a way to retrieve the points with this method?
Thank you!

**Comment by TFArkS** [comment]
It would be more computationally efficient if the function returned these values, but a workaround is running the following on each returned bounding box:
obb.get_point_indices_within_bounding_box(pcd.points)
